packages/create-sitecore-jss/src/templates/nextjs/package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.1",
     "eslint-plugin-yaml": "^0.5.0",
-    "graphql-let": "^0.18.6",
+    "graphql-let": "0.18.4",
```

Comment:
Is there a reason to "fix" this version?
In general we need to use the "latest" version even if a vulnerability is not fixed there. In a such case it's a problem on 3rd party lib side

Reply:
yes, latest graphgl-let relies on a fixed vulnerable version of minimist and this is why I downgraded it; I will revert that to point to latest.